ontology/yaml/resources/states/states.yaml

```diff
@@ -90,3 +90,9 @@ SOFTSTART: "Mode of operation of rectifier where the input power supply is gradu
 NORMAL: "Function, state or flow which is an ideal or expected condition."
 REVERSED: "Function, state or flow which is opposed to the expected or normal condition."
 
+# FOR HEAT RECOVERY UNIT
+STARTUP: "Operation mode where unit is starting up."
+WARMUP: "Operation mode where unit is warming up."
+OVERRUN: ""
+INTERMITTENT: ""
+FIREMODE: "Operation mode during fire alarm."
```

Comment:
Would this be a separate operating mode, or would it just be `OFF` when the `fire_alarm` field is returning `ACTIVE`?